benchmarks/dynamo/common.py

```diff
@@ -354,6 +354,19 @@ def deterministic_torch_manual_seed(*args, **kwargs):
     torch.manual_seed = deterministic_torch_manual_seed
 
 
+def empty_gpu_cache(device):
+    """
+    Explicitly empty gpu cache to avoid OOM in subsequent run.
+    """
+    assert (
+        device != ""
+    ), "The empty_gpu_cache needs to be called with a non empty device str"
+    if device == "cuda":
+        torch.cuda.empty_cache()
+    if device == "xpu":
```

Comment:
Nit: elif

Reply:
Thanks for the review! Solved in https://github.com/pytorch/pytorch/pull/126377/commits/56a7d2ea7b9d571268396d21c95691b92bcccf73